core/src/Slay/Prim.hs

```diff
@@ -82,6 +86,18 @@ data PrimCurve g =
       curveDirection :: g Direction
     }
 
+
+-- | circle adt
+data PrimCircle g = Circle
+    { circleColor :: g (Maybe Color)
+    , circleCenterX :: Double
+    , circleCenterY :: Double
+    , circleRadius :: Double
+    }
+
+
+
+-- | Inj typeclass
```

Comment:
Once again, the comment "Inj typeclass" doesn't say anything new. I can see `class Inj` on the next line.